include/tester/vtcp/VTCPSession.hpp

```diff
@@ -30,24 +30,24 @@ class VTCPSession : public TCPSession
         }
         catch(const std::exception& e)
         {
-            std::cout << e.what() << std::endl;
+            std::cerr << e.what() << std::endl;
         }
     }
 
     ~VTCPSession() = default;
 
-    void onVtcpConnect(const Message & data)
+    void onVtcpConnect(const Message & request)
     {
         std::cout << "New client connected" << std::endl;
-        Message result;
-        result.push(static_cast<int>(VTCPOpcode::VTCP_CONNECT_RES));
+        Message response;
+        response.push(static_cast<int>(VTCPOpcode::VTCP_CONNECT_RES));
         std::string result_msg = "Hello new client";
-        result.push(result_msg);
-        result.pushSize();
-        send(result.getPayload());
+        response.push(result_msg);
+        response.pushSize();
```

Comment:
rename `pushSize` to `setPayloadSize`, I think this should be a private method and be called after each `push`, but I'll inform you regarding this tomorrow.